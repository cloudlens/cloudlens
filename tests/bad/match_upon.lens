process upon (hi){
    print(e.hi)
}
