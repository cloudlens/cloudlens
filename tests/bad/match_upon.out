Syntax Error: tests/bad/match_upon.lens line 1 character 8
no viable alternative at input 'upon'
